zipline/finance/performance/tracker.py

```diff
@@ -350,10 +362,13 @@ def handle_minute_close(self, dt, data_portal):
         # cumulative returns
         bench_since_open = (1. + bench_returns).prod() - 1
 
+        # Do not calculate expected shortfall every minute as it requires a
```

Comment:
If feels odd to me that all the other risk metrics are managed by the RiskMetricsCumulative class, but then `expected_shortfall` is passed in as a parameter.